src/test/moves/baton_pass.test.ts

```diff
@@ -45,24 +45,27 @@ describe("Moves - Baton Pass", () => {
     // round 1 - buff
     game.doAttack(getMovePosition(game.scene, 0, Moves.NASTY_PLOT));
     await game.toNextTurn();
-    expect(game.scene.getPlayerPokemon()!.summonData.battleStats[BattleStat.SPATK]).toEqual(2);
+
+    let playerPokemon = game.scene.getPlayerPokemon()!;
+
+    expect(playerPokemon.getStatStage(Stat.SPATK)).toEqual(2);
 
     // round 2 - baton pass
     game.doAttack(getMovePosition(game.scene, 0, Moves.BATON_PASS));
     game.doSelectPartyPokemon(1);
     await game.phaseInterceptor.to(TurnEndPhase);
 
     // assert
-    const playerPkm = game.scene.getPlayerPokemon()!;
-    expect(playerPkm.species.speciesId).toEqual(Species.SHUCKLE);
-    expect(playerPkm.summonData.battleStats[BattleStat.SPATK]).toEqual(2);
+    playerPokemon = game.scene.getPlayerPokemon()!;
+    expect(playerPokemon.species.speciesId).toEqual(Species.SHUCKLE);
+    expect(playerPokemon.getStatStage(Stat.SPATK)).toEqual(2);
   }, 20000);
 
   it("passes stat stage buffs when AI uses it", async() => {
     // arrange
     game.override
       .startingWave(5)
-      .enemyMoveset(new Array(4).fill([Moves.NASTY_PLOT]));
+      .enemyMoveset([ Moves.NASTY_PLOT, Moves.NASTY_PLOT, Moves.NASTY_PLOT, Moves.NASTY_PLOT ]);
```

Comment:
```suggestion
      .enemyMoveset(new Array(4).fill([Moves.NASTY_PLOT]);
```
NIT: Reverting change